ass2-task/sass/pages/_home.scss

```diff
@@ -0,0 +1,786 @@
+        height: 370px;
+        width: 100%;
+      }
+    }
+
+    .map-analysis {
+      background-color: $green;
+      float: left;
+      width: 30%;
+
+      .map-data {
+        background-color: $transparent;
+        color: $white;
+        margin: 30px 0 35px;
+        text-align: center;
+
+        i {
+          font-size: 30px;
+        }
+
+        h3 {
+          font-size: $font-16;
+          margin: 5px 0;
+        }
+
+        h2 {
+          font-size: $font-16;
+          font-weight: 300;
+          text-transform: uppercase;
+        }
+      }
+    }
+
+    .visitors-data {
+      background-color: $transparent;
+      float: left;
+      height: 100%;
+      margin-top: 20px;
+      width: 40%;
+
+      .data-details {
+        color: $white;
+        height: 100px;
+        margin: 0 0 10px 20px;
+
+        .data-image {
+          float: left;
+          height: 100px;
+          line-height: 100px;
+          position: relative;
+          text-align: center;
+          width: 30%;
+
+          i {
+            font-size: $font-40;
+            margin-top: 30px;
+          }
+        }
+
+        .overlay {
+          background-color: $light-black;
+          bottom: 0;
+          height: 100px;
+          left: 0;
+          position: absolute;
+          right: 0;
+          top: 0;
+        }
+
+        .data-report {
+          float: left;
+          padding: 10px;
+          width: 70%;
+
+          h3 {
+            font-size: $font-14;
+            font-weight: 300;
+            margin: 5px 0;
+            text-transform: uppercase;
+          }
+
+          .data-progress-bar {
+            background-color: $border-grey;
+            height: 5px;
+            line-height: 5px;
+            margin-bottom: 10px;
+            width: 100%;
+
+            div {
+              background-color: $white;
+              height: 5px;
+              line-height: 5px;
+              width: 70%;
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+//===============dashboard ends======//
+
+//===============media queries======//
+@media only screen and (min-width:1024px) and (max-width:1440px) {
+  .dashboard-container {
+    .dashboard-analysis {
+
+      .analysis-value {
+        h4 {
+          font-size: $font-14;
+        }
+      }
+    }
+  }
+}
+
+@media only screen and (min-width:768px) and (max-width:1023px) {
+  .sidebar {
+    display: none;
+  }
+
+  .dashboard-container {
+    margin-left: 0;
+    width: 100%;
+
+    .dashboard-analysis {
+      width: 22%;
+
+      .analysis-value {
+        h4 {
+          font-size: $font-14;
+        }
+      }
+    }
+  }
+}
+
+
+@media only screen and (min-width:768px) and (max-width:810px) {
+  .dashboard-container {
+    .dashboard-analysis {
+      width: 21%;
+
+      .analysis-value {
+        h4 {
+          font-size: $font-14;
+        }
+      }
+    }
+  }
+}
+
+@media only screen and (min-width:320px) and (max-width:767px) {
```

Comment:
@priyanka0901 You can write this media query like this also 
`@media only screen and (max-with:767px) {}`